src/Sylius/Bundle/AdminBundle/Resources/views/ProductAttribute/_form.html.twig

```diff
@@ -0,0 +1,28 @@
+<div class="ui two column stackable grid">
+    {{ form_errors(form) }}
+    <div class="column">
+        <div class="ui segment">
+            <div class="two fields">
+                {{ form_row(form.code) }}
+                {{ form_row(form.type) }}
+            </div>
+            <h4 class="ui dividing header">{{ 'sylius.ui.configuration'|trans }}</h4>
+            <div class="ui segment configuration">
+                {% if form.configuration is defined %}
+                    {% for field in form.configuration %}
```

Comment:
Maybe you could use ``{% for field in form.configuration if form.configuration is defined %}``?